### PR TITLE
Feature/5 store changes

### PR DESCRIPTION
**Pull request by drewbo**
Minimal implementation of #4/#5

It currently doesn't track individual vertices while a line is drawn but does track the history of added lines, deleted lines, and lines edited creation. The first feature is waiting on https://github.com/mapbox/mapbox-gl-draw/pull/599.

Also adds `redux-logger` and `geojson-random` (forked) for debugging help

### CODE REVIEW

**Comment by dereklieu** [comment]
Notes copied from Slack

>derek
@drew so just making sure I have this right: for undo, the flow is `UI -> dispatch(undo()) -> store -> componentWillReceiveProps -> dispatch(completeUndo())`

>drew
exactly
the `componentWillReceiveProps` part syncs the store with the Draw.store, the completion part removes that from the "present" so it doesn't sync again and keeps the history in order

>derek
and is `feature.undo` and `feature.redo` an internal representation that gl draws knows about?
referring to https://github.com/developmentseed/skynet-scrub/pull/6/files#diff-479d1a4f9477a444f1205031f44e5332R77

>drew
no sorry that part is poorly document, draw has no idea about undo/redo
basically, when we store something in history, we need to know how to undo or redo it, so we store both halves of the act in the history: https://github.com/developmentseed/skynet-scrub/pull/6/files#diff-479d1a4f9477a444f1205031f44e5332R39
then when we are replaying it (either forwards or backwards), we just grab the right one depending upon which direction we are going in

>derek
ah i see. so `undo` or `redo` are just the geojson definition of the feature
that part wasn’t clear to me
but it makes sense

>drew
yeah, add needs "empty" + the added feature, delete needs the opposite, updated needs the old and new state
